haiku-apps/bephonebook/bephonebook-1.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A phone book manager"
+DESCRIPTION="BePhoneBook is a colorful phonebook made to help you stay \
+organized with your family's or friends' phone numbers."
+HOMEPAGE="https://github.com/HaikuArchives/BePhoneBook"
+COPYRIGHT="1999 Michael Melloff \
+	Fabien Fulhaber"
+LICENSE="BePhoneBook"
```

Comment:
Please add the license text to `licenses/libsanta` relative to the recipe and change `LICENSES` to `libsanta`.

While a lot of the code uses the linked license, the main files are **not** covered by this license, so you'd need to add `Unknown` there.